### DISCUSSION TITLE
get status of previous step execution

### DISCUSSION THREAD

**Discussion by sarut0bi** [getgauge/gauge#2043] [General]
Hi ,
Is there anyway to get previous step result (if it failed or not) in `gauge.hooks.afterStep(fn, [opts]) { ... }` function?

**Comment by sriv** [comment]
Hi, you can use the `context` that gets passed into the execution hooks. Ex:

```
beforeScenario(function (context) {
  console.log(context.currentStep && context.currentStep.isFailed);
 //...
});
```

**Comment by sarut0bi** [comment]
Exactly what i need ty !
